etna/models/nn/tft_native/tft.py

```diff
@@ -0,0 +1,647 @@
+                sample["static_reals"][feature] = (
+                    values[column_to_index[feature]][:1].reshape(-1, 1).astype(float)
+                )  # (1, 1)
+
+            for feature in self.static_categoricals:
+                sample["static_categoricals"][feature] = (
+                    values[column_to_index[feature]][:1].reshape(-1, 1).astype(float)
+                )  # (1, 1)
+
+            for feature in self.time_varying_categoricals_encoder:
+                sample["time_varying_categoricals_encoder"][feature] = (
+                    values[column_to_index[feature]][start_idx : start_idx + encoder_length]
+                    .reshape(-1, 1)
+                    .astype(float)
+                )  # (encoder_length, 1)
+
+            for feature in self.time_varying_categoricals_decoder:
+                sample["time_varying_categoricals_decoder"][feature] = (
+                    values[column_to_index[feature]][start_idx + encoder_length : start_idx + total_sample_length]
+                    .reshape(-1, 1)
+                    .astype(float)
+                )  # (decoder_length, 1)
+
+            for feature in self.time_varying_reals_encoder:
+                sample["time_varying_reals_encoder"][feature] = (
+                    values[column_to_index[feature]][start_idx : start_idx + encoder_length]
+                    .reshape(-1, 1)
+                    .astype(float)
+                )  # (encoder_length, 1)
+
+            for feature in self.time_varying_reals_decoder:
+                sample["time_varying_reals_decoder"][feature] = (
+                    values[column_to_index[feature]][start_idx + encoder_length : start_idx + total_sample_length]
+                    .reshape(-1, 1)
+                    .astype(float)
+                )  # (decoder_length, 1)
+
+            return sample
+
+        start_idx = 0
+        while True:
+            batch = _make(
+                values=values,
+                column_to_index=column_to_index,
+                segment=segment,
+                start_idx=start_idx,
+                encoder_length=encoder_length,
+                decoder_length=decoder_length,
+            )
+            if batch is None:
+                break
+            yield batch
+            start_idx += 1
+
+    def configure_optimizers(self) -> "torch.optim.Optimizer":
+        """Optimizer configuration."""
+        optimizer = torch.optim.Adam(self.parameters(), lr=self.lr, **self.optimizer_params)
+        return optimizer
+
+
+class TFTNativeModel(DeepBaseModel):
+    """TFT model. For more details read the `paper <https://arxiv.org/abs/1912.09363>`_.
+
+    Model needs label encoded inputs for categorical features, for that purposes use :py:class:`~etna.transforms.LabelEncoderTransform`.
+    Feature values that were not seen during `fit` should be set to NaN for expected behaviour with `strategy="none"`
+
+    Passed features for each feature type are not validated.
```

Comment:
This is difficult to read and isn't specific at all. We should write smth more specific like: `Passed feature values aren't validated on being static or being correctly label encoded.`.